Python/ceval.c

```diff
@@ -3174,9 +3174,10 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, PyFrameObject *f, int throwflag)
                     la = &co_opcache->u.la;
                     if (la->type == type && la->tp_version_tag == type->tp_version_tag)
                     {
-                        // Hint >= 0 is a dict index; hint < 0 is an inverted slot index.
-                        if (la->hint < 0) {
-                            /* Even faster path -- slot hint */
+                        // Hint >= 0 is a dict index; hint == -1 is a dict miss.
```

Comment:
Oh, right! How did I miss that on review :/

Reply:
Same way I missed it writing the code. :-(

This code is hairy.